gkShell/docker-compose.yml

```diff
@@ -44,7 +44,9 @@ services:
         environment:
```

Comment:
This file is not used anymore in new deployments. may be safely removed.

Reply:
I fact in #496 I hesitated in : mark gkShall as deprecated/no more support and keep file/doc onto gkShell subdirectory (for any windows 7 contributor , if any ... :p) OR remove completely gkShell.

I think I will finish to migrate gkShell sh feature, then remove all (out of objection). But that's part of 496 I can do after merge